addons/compact-messages/addon.json

```diff
@@ -5,14 +5,24 @@
     {
       "name": "yes-i-am-lanky",
       "link": "https://scratch.mit.edu/users/yes-i-am-lanky/"
+    },
+    {
+      "name": "DNin01",
+      "link": "https://github.com/DNin01"
     }
   ],
   "settings": [
     {
       "name": "Hide message type icons",
       "id": "hide_icons",
       "type": "boolean",
-      "default": true
+      "default": false
```

Comment:
Forgot to mention this, but is it okay to change this default to `false`? Turning this setting off doesn't really waste any space unless anything word-wraps on the Messages page.

Reply:
If your new remove borders setting is enabled by default, yes, otherwise I don't think it really matters.

Reply:
Sounds okay to me